guide/src/tutorials/draw/drawing-images.md

```diff
@@ -0,0 +1,188 @@
+# Drawing Images
+
+**Tutorial Info**
+
+- Author: Josiah Savary
+- Required Knowledge:
+    - [Getting Started](/getting_started.md)
+    - [Draw a Sketch](./draw-a-sketch.md)
+- Reading Time: 20 minutes
+
+---
+
+In this tutorial, we explore drawing images with nannou. We will cover loading textures from images the app's assets directory and drawing them, and applying basic transformations.
+
+You can also find this file, and other useful examples, in the [examples](https://github.com/nannou-org/nannou/tree/master/examples) directory of the nannou source repository.
+
+To begin with, we will need a nannou project file to work with. Copy the following into new file:
+
+```rust,no_run
```

Comment:
Perhaps it was just me, but this code block failed to run... could it be combined with the following code block as the only difference is in the model method? If not, we should mention that this section won't run.

Reply:
Yeah that's a good call! I'll start out with bootstrapping the assets dir and move this part into the following code block.